Assignment 1/src/main/java/com/nuclei/assignment/service/tax/ItemTaxImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.nuclei.assignment.service.tax;
+
+import com.nuclei.assignment.constants.ItemTaxConstantsUtils;
+
+/**
+ * Item Tax interface for different types of Item Type.
+ */
+public class ItemTaxImpl implements ItemTax {
+  
+  @Override
+  // Calculation of tax for ItemType.RAW
```

Comment:
these comments here are not required, name of the function is suffice.

Reply:
done